src/mutation.cpp

```diff
@@ -982,27 +1025,30 @@ bool Character::mutation_ok( const trait_id &mutation, bool allow_good, bool all
     return true;
 }
 
-bool Character::roll_bad_mutation() const
+bool Character::roll_bad_mutation( const mutation_category_id &categ ) const
 {
+    // we will never have worse odds than this no matter our instability
+    float MAX_BAD_CHANCE = 0.67;
```

Comment:
\> Is there any method that in-repo mods can hook into for changing the effective instability?
Thinking how it can be implemented - the easiest possibility i can see is add enchantment that modifies MAX_BAD_CHANCE, but it honesly pretty limited, and there is no effect if one would try to increase it
```
    float MAX_BAD_CHANCE = enchantment_cache->modify_value( enchant_vals::mod::MUTATION_MAX_BAD_CHANCE_MOD, 0.67 );
```

Reply:
yeah that can probably be replaced with something, somewhere, eventually. i just wanted to implement the lazy solution first lol